docs/Configuration-Settings.md

```diff
@@ -157,6 +157,11 @@ number.
   <td>The externally visible hostname of Tachyon's master address.</td>
 </tr>
 <tr>
+  <td>tachyon.master.hostname.listening</td>
+  <td></td>
+  <td>(optional) The address the master will listen on. If set to the wildcard address, "*", the master will listen on all addresses. If unspecified, the master will listen on the address specified for `tachyon.master.hostname`.</td>
```

Comment:
Hrmmm, that is a good question. Technically, the option is not specified by default, which is why I left it empty.

However, because it is unspecified, the master will listen on tachyon.master.hostname. Does the "default" column mean the default set value, or the resulting behavior?